src/pages/HttpBin/index.js

```diff
@@ -1,7 +1,146 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import {
+  Container,
+  Grid,
+  Paper,
+  Button,
+  Typography,
+  CircularProgress,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import axios from 'axios';
+
+import FormInput from '../../components/FormInput';
+import SelectFormInput from '../../components/SelectFormInput';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    maxWidth: '1360px',
+    padding: '10px',
+    margin: '0 auto',
+    boxShadow: 'none',
+  },
+  sendButton: {
+    width: '100%',
+    marginTop: '14px',
+    height: '42px',
+  },
+}));
+
+// ----------------------- EXAMPLES
+const exampleJson = `{"name":"Molecule Man","age":29,"secretIdentity":"Dan Jukes","powers":["Radiation resistance","Turning tiny","Radiation blast"]}`;
+// --------------------------END EXAMPLES
 
 const HttpBin = () => {
-  return <h1>HttpBin</h1>;
+  const classes = useStyles();
+  const [methodType, setCurrency] = useState('GET');
+  const [queryParameters, setQueryParameters] = useState('get');
+  const [bodyParameters, setBodyParameters] = useState(exampleJson);
+  const [response, setResponse] = useState({});
+  const [responseIsLoad, setResponseIsLoad] = useState(true);
+
+  const createRequest = () => {
+    const axiosConfig = {
+      method: methodType,
+      baseURL: `https://httpbin.org/`,
+      data: JSON.parse(bodyParameters),
```

Comment:
won't it raise an exception when JSON string is broken?

Reply:
update in next commit